src/app/pages/bugs/bugs.component.scss

```diff
@@ -0,0 +1,37 @@
+.container{
+    display:flex;
+    // flex-wrap: wrap;
+    align-items: stretch;
+    margin-top:15px;
+}
+
+.box{
+    font-size:15px;
+    padding:18px;
+    flex-grow: 4;
+    color: white;
+    font-weight: 400;
+    text-align: center;
+    cursor: pointer;
+}
+
+.box:hover {
+    background-color: olive;
+}
+
+.box-details {
+    font-size:15px;
+    padding:15px;
+    color: white;
+    font-weight: 400;
+    text-align: center;  
+    // flex: 1 1 0px;
+    flex-grow:0 0 50%;
+    width: 50%;
+    color: black;
+}
+
+.box1{background:mediumseagreen;}
+.box2{background:darkcyan;}
+.box3{background:steelblue;}
+.box4{background:darkkhaki;}
```

Comment:
can we remove the commented code and put in a space after colon?

Reply:
yes commented lines are removed